app/assets/stylesheets/application.css

```
@@ -13,3 +13,133 @@
  *= require_tree .
  *= require_self
  */
```

If you load the trix editor styles in `application.css` and delete the `actiontext.scss` file, your custom styles should no longer be overwritten.
```suggestion
 *= require_tree .
 *= require_self
 *= require trix/dist/trix
 */
```

---

Thank you! This worked.